app/manifest/v3/chrome.json

```diff
@@ -1,6 +1,6 @@
 {
   "content_security_policy": {
-    "extension_pages": "default-src 'self'; frame-ancestors 'none'"
+    "extension_pages": "script-src 'self'; object-src 'self';"
```

Comment:
@Gudahtt can you verify that this change is inline with recent changes you made?

Reply:
Now that `frame-ancestors` has been restored, yes. `default-src` was only added to prevent a _different_ crash. Changing it to `script-src 'self'; object-src 'self';` is fine, it also prevents the same crash while fixing the one described above.

For context, we found that `script-src 'self'; object-src 'self';` is [the default CSP used by Chromium for MV3 extensions](https://source.chromium.org/chromium/chromium/src/+/main:extensions/common/manifest_handlers/csp_info.cc;l=42;drc=9e249731c0b1aab0e53fb776b9c11e673e721c45?q=default_isolated_world_csp&ss=chromium).